src/Components/CategoriesNew.js

```diff
@@ -8,10 +8,10 @@ import { createCategory } from "../Services/Organizations/CategoriesService";
 export default function CategoriesNew(props) {
   const [name, setName] = useState("");
   const { onClose } = props;
-  const [organizationId, setOrganizationId] = useState("");
-  const [errors, setErrors] = useState([]);
-  const [loading, setLoading] = useState(true);
-  const [isHiddenNew, setIsHiddenNew] = useState(true);
+  // const [organizationId, setOrganizationId] = useState("");
+  // const [errors, setErrors] = useState([]);
+  // const [loading, setLoading] = useState(true);
+  // const [isHiddenNew, setIsHiddenNew] = useState(true);
```

Comment:
Can we delete this or will they be needed soon?

Reply:
I was thinking we could leave them as placeholders for the next step of implementation - if it's confusing we can delete.